src/main/java/com/epam/preprod/sirenko/Demo.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.preprod.sirenko;
+
+import com.epam.preprod.sirenko.containers.CommandContainer;
+import com.epam.preprod.sirenko.dao.ProductDAO;
+
+import java.io.*;
+
+public class Demo {
+	CommandContainer commandContainer;
```

Comment:
We dont need it here as field,we can just create all we need for demonstration in psvm

Reply:
deleted